ui/app/AppLayouts/Wallet/Components/TokenSettingsModalContent.qml

```diff
@@ -54,10 +54,13 @@ Item {
         anchors.left: parent.left
         anchors.leftMargin: 32
 
-        SearchBox {
+        Input {
```

Comment:
I wonder, maybe we want to keep the SearchBox component and make it very simple using the Input component? Since we use it twice, do you think it's worth it?

Reply:
yah, not a bad idea. I added the component back, but this time inside the shared folder